README.md

```diff
@@ -356,6 +360,10 @@ newman.run({
         "_postman_exported_at": "2016-10-17T14:26:34.940Z",
         "_postman_exported_using": "Postman/4.8.0"
     }
+    envVar: [ 
```

Comment:
This was the only place I could find an example of how to use envVar.  It would be nice to get this included in the documentation.

Reply:
This merged PR is for the documentation...

![image](https://user-images.githubusercontent.com/12074633/95385975-afb9df00-08bc-11eb-937a-4324ad6cbe38.png)